lab_1/start.py

```diff
@@ -15,11 +17,26 @@
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'de.txt'), 'r', encoding='utf-8') as file_to_read:
         de_text = file_to_read.read()
 
+    with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'la.txt'), 'r', encoding='utf-8') as file_to_read:
+        la_text = file_to_read.read()
+
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'unknown.txt'), 'r', encoding='utf-8') as \
             file_to_read:
         unknown_text = file_to_read.read()
 
+    en_profile = main.load_profile(os.path.join(PATH_TO_PROFILES, 'en.json'))
+    de_profile = main.load_profile(os.path.join(PATH_TO_PROFILES, 'de.json'))
+    la_profile = main.load_profile(os.path.join(PATH_TO_PROFILES, 'la.json'))
+    profiles = [en_profile, de_profile, la_profile]
+
+    unknown_profile = main.create_language_profile('unk', unknown_text, [])
+
+    detection = main.detect_language_advanced(unknown_profile, profiles, [], 6)
+
+    main.save_profile(unknown_profile)
```

Comment:
can you describe in writing what happens in` start.py`?

Reply:
Firstly, we have to get profiles of known languages. We can load language profiles from the files in folder "profiles", using load_profile from main.py (as I did), or we can read the files with the texts from the folder "texts" and create profiles of known languages, using function create_language_profile from main.py (I decided not to do it, because it's redundant, I just kept the piece of code with reading texts, because it was here initially).
After that, we create a list with the profiles of known languages and create the profile of unknown language, using function create_language_profile from main.py. Then we detect this language, using function detect_language_advanced. Also, we write the profile of unknown language to file, using save_profile. Finally, using "assert", we check whether the result is correct.